person.rb

```diff
@@ -1,15 +1,20 @@
 # Defines a person class
-class Person
+class Person < Nameable
   attr_reader :id
```

Comment:
Same here, Please require the `Nameable` class from the `nameable.rb` file using `require_relative` to avoid the **NameError**.